### PR TITLE
docs: improve docs on TLD info for connection

### PR DESCRIPTION
**Pull request by Tosta-Mista**
Hello, 

This PR to add new documentation examples. 

Issue: #118

### CODE REVIEW

**Review by pacificcode — APPROVED** [review]
LGTM

File: docs/configure.md [hunk 1]
```diff
@@ -37,7 +39,8 @@ You can place your temporary config in `.cache/credentials.json` as this is igno
       "clientId": "",
       "clientSecret": "xxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxx-xxxxx"
     },
-    "tenant": "mytenant"
+    "tenant": "mytenant",
+    "tld": "eu"
```
Comment (sheldonhull) [inline docs/configure.md:43]: Can you revert the changes for the code sample since it's already included in the linked docs? Appreciate it!

**Review by sheldonhull — CHANGES_REQUESTED** [review]
minor quick changes. greatly appreciate the contribution!

**Comment by sheldonhull** [comment]
to avoid delay/reworking, I went ahead and got the change in, so definitely appreciate bringing up that it wasn't clear enough. cheers!

**Pull request closed without merging**